pkg/controller/core/queuedworkload_controller.go

```diff
@@ -128,7 +128,7 @@ func (r *QueuedWorkloadReconciler) Update(e event.UpdateEvent) bool {
 		log = log.WithValues("clusterQueue", wl.Spec.Admission.ClusterQueue)
```

Comment:
let's use `klog.KRef` while you are at it.

Reply:
maybe another PR? this would mean to mod also somethings on the lines above

Reply:
```bash
cannot use klog.KRef("prevStatus", prevStatus) (value of type klog.ObjectRef) as logr.Logger value in assignmentcompiler[IncompatibleAssign](https://pkg.go.dev/golang.org/x/tools/internal/typesinternal?utm_source%3Dgopls#IncompatibleAssign)
```
this is why this needs to be a full PR just for that change

Reply:
sure, separate PR is fine.